src/Core/Translation/Builder/Map/Domain.php

```diff
@@ -207,8 +207,9 @@ public function mergeTree(array &$tree): array
      */
     public static function splitDomain(string $domain): array
     {
+        $inflector = InflectorFactory::create()->build();
```

Comment:
Move it to constructor

Reply:
A last one @atomiix

Reply:
It's a static method, I can do a singleton but I'm not sure it is worth it?

Reply:
Why not by this way: https://www.doctrine-project.org/projects/doctrine-inflector/en/2.0/index.html#no-operation-inflector ?